OCKSample/MainTabs/MainView.swift

```diff
@@ -49,6 +49,20 @@ struct MainView: View {
                         .navigationBarTitle("")
                         .navigationBarHidden(true)
 
+                    InsightsView()
+                        .tabItem {
+                            if selectedTab == 0 {
+                                Image(systemName: "chart.pie.fill")
```

Comment:
Image's for your `tabItem` can be any you want. Feel free to change any of them to images you like better.